src/main/resources/conf/fdpConfig.yml

```diff
@@ -16,12 +16,12 @@ store:
 urlPath:
     root: /fdp
 metadataProperties:
-    rootSpecs: http://rdf.biosemantics.org/fdp/shex/fdpMetadata
-    catalogSpecs: http://rdf.biosemantics.org/fdp/shex/catalogMetadata
-    datasetSpecs: http://rdf.biosemantics.org/fdp/shex/datasetMetadata
-    distributionSpecs: http://rdf.biosemantics.org/fdp/shex/distributionMetadata   
-    publisherURI: http://dtls.nl
-    publisherName: dtls
+    rootSpecs: https://www.purl.org/fairtools/fdp/schema/0.1/fdpMetadata
```

Comment:
Just a question: is it common to put the `www.` subdomain on a purl.org url? I don't recall seeing it before. It's totally valid of course, but if no one is ever going to use it we might as well leave it out.

Reply:
It works without `www.` but we used `www.` in the shex's base uri